### PR TITLE
fix(es/minifier): Enable seq inliner for const declarations

### PR DESCRIPTION
**Pull request by Austaras**
**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: crates/swc/tests/fixture/issues-5xxx/5112/output/index.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 function c(r, n) {
-    let e = new Uint8Array(4 * r * r);
-    return {
-        e,
+    return new Uint8Array(4 * r * r), {
```
Comment (kdy1) [inline crates/swc/tests/fixture/issues-5xxx/5112/output/index.js:2]: This seems wrong

Comment (Austaras) [reply crates/swc/tests/fixture/issues-5xxx/5112/output/index.js]: This is wrong. It's caused by `unused: false`. However after fix, it's still wrong -- under `unused: false` var declaration should not be drop, but that's another (low proority) issue.

File: crates/swc/tests/tsc-references/exportAssignmentOfExportNamespaceWithDefault.2.minified.js [hunk 2]
```diff
@@ -2,7 +2,5 @@
 // https://github.com/microsoft/TypeScript/issues/39149
 Object.defineProperty(exports, "__esModule", {
     value: !0
-});
-const _interop_require_default = require("@swc/helpers/_/_interop_require_default"), _a = /*#__PURE__*/ _interop_require_default._(require("a"));
-(0, _a.default)();
+}), (0, /*#__PURE__*/ require("@swc/helpers/_/_interop_require_default")._(require("a")).default)();
```
Comment (kdy1) [inline crates/swc/tests/tsc-references/exportAssignmentOfExportNamespaceWithDefault.2.minified.js:5]: This is different from the original code, but it looks like a bugfix.

File: crates/swc_ecma_minifier/tests/benches-full/d3.js [hunk 3]
```diff
@@ -1381,7 +1381,7 @@
             return (1 === (a = isNaN(a) ? 1 : Math.max(0, Math.min(1, a))) ? "hsl(" : "hsla(") + (this.h || 0) + ", " + 100 * (this.s || 0) + "%, " + 100 * (this.l || 0) + "%" + (1 === a ? ")" : ", " + a + ")");
         }
     }));
-    const radians = Math.PI / 180, degrees = 180 / Math.PI, t0 = 4 / 29, t1 = 6 / 29, t2 = 3 * t1 * t1, t3 = t1 * t1 * t1;
+    const radians = Math.PI / 180, degrees = 180 / Math.PI, t0 = 4 / 29, t1 = 6 / 29, t2 = 3 * (6 / 29) * (6 / 29), t3 = 6 / 29 * (6 / 29) * (6 / 29);
```
Comment (kdy1) [inline crates/swc_ecma_minifier/tests/benches-full/d3.js:1384]: This will degrade performance (although it's very minor)

Comment (Austaras) [reply crates/swc_ecma_minifier/tests/benches-full/d3.js:1384]: Yes, but this is well beyond the scope of this PR.

File: crates/swc/tests/tsc-references/exportAssignmentOfExportNamespaceWithDefault.2.minified.js (hunk 2 above)
Comment (Austaras) [reply crates/swc/tests/tsc-references/exportAssignmentOfExportNamespaceWithDefault.2.minified.js:5]: This is the same as old code. both `_interop_require_default` and `_a` are inlined.

File: crates/swc_ecma_minifier/tests/benches-full/d3.js (hunk 3 above)
Comment (kdy1) [reply crates/swc_ecma_minifier/tests/benches-full/d3.js:1384]: Make sense. Thanks!

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_ecma_minifier

**Comment by kdy1** [comment]
Can you update the test references? (I think those are unrelated to this PR, though)

**Review by kdy1 — APPROVED** [review]

**Pull request merged**